AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -1694,6 +1694,7 @@ goodreads.com##.siteHeader__topFullImage
 skysmart.ru##ssm-articles-another-product-banner
 deepl.com##div[class*="write_ad_dialog--"]
 dwdisc.com##.btn-prem
+||tuber-ch.com/static/img/coffee.png
```

Comment:
Where does this request occur? I don't see.

Reply:
It displayed on top like a screenshot.
But it seems that it was not just one type.
Should I hide the place instead of the image?

Reply:
Is this now `tuber-ch.com/static/img/belta.png`? If so, I don't think it should be default-blocked. May possibly be annoyance, but only if we get real user complaint. But we can add removeparam rules
```
||belta.co.jp^$removeparam=cid
||belta.co.jp^$removeparam=urrad
```
(not sure about `p` parameter)

Reply:
Yes, it is.
I will only add rule that you suggested.

Reply:
Then please add this PR as comment in URL tracking filter.

Reply:
OK, Added.